docs/www/kubernetes-external-connect.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Connect to a Kubernetes cluster from an external client
+order: 0
+---
+
+# Connect to a Kubernetes cluster from an external client
+
+The [Kubernetes Quick Start guide](quick-start-kubernetes.md) describes how to quickly get up and running with a Kubernetes cluster.
+Those instructions only provide access to the cluster from within the Kuberenetes network.
+
+Here we'll show you an example of how to set up Kubernetes in [Google GKE](https://cloud.google.com/kubernetes-engine) or [Amazon EKS](https://aws.amazon.com/eks)
+so you can work with Redpanda from outside of the Kubernetes network.
+
+Let's get started...
+
+## Create a Kubernetes cluster
+
+1. Create three node cluster:
+
+<tabs>
+
+  <tab id="AWS EKS">
+
+  Use the [EKS Getting Started](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) guide to set up EKS.
+  When you finish, you'll have `eksctl` installed so that you can create and delete clusters in EKS.
+  Then, create an EKS cluster with:
+
+  ```
+  eksctl create cluster \
+  --name redpanda \
+  --nodegroup-name standard-workers \
+  --node-type m5.xlarge \
+  --nodes 3 \
+  --nodes-min 1 \
+  --nodes-max 4 \
+  --node-ami auto
+  ```
+
+  It will take about 10-15 minutes for the process to finish.
+
+  </tab>
+
+  <tab id="Google GKE">
+
+  First complete the "Before You Begin" steps described in [Google Kubernetes Engine Quickstart](https://cloud.google.com/kubernetes-engine/docs/quickstart).
+  Then, create a cluster with:
+
+  ```
+  gcloud container clusters create redpanda --machine-type e2-standard-4 --cluster-version 1.19 && \
+  gcloud container clusters get-credentials redpanda
+  ```
+
+  **_Note_** - You may need to add a `--region`, `--zone`, or `--project` to this command.
+
+  </tab>
+</tabs>
+
+2. Install cert-manager
+
+  ```bash
+  helm repo add jetstack https://charts.jetstack.io && \
+  helm repo update && \
+  helm install \
+    cert-manager jetstack/cert-manager \
+    --namespace cert-manager \
+    --create-namespace \
+    --version v1.2.0 \
+    --set installCRDs=true
+  ```
+
+3. Install the latest redpanda operator:
+
+  ```
+  kubectl apply -k https://github.com/vectorizedio/redpanda/src/go/k8s/config/default
+  ```
+
+4. Install a cluster with external connectivity:
+
+  ```
+  kubectl apply -f https://raw.githubusercontent.com/vectorizedio/redpanda/dev/src/go/k8s/config/samples/external_connectivity.yaml
+  ```
+
+5. Open the firewall for access to the cluster:
```

Comment:
This section can points that only Google cloud need this acction. Digital Ocean and AWS EKS are open by default.